bin/reth/src/node/mod.rs

```diff
@@ -120,7 +124,13 @@ impl Command {
         let genesis_hash = init_genesis(db.clone(), self.chain.genesis.clone())?;
 
         let network = config
-            .network_config(db.clone(), chain_id, genesis_hash, self.network.disable_discovery)
+            .network_config(
+                db.clone(),
+                chain_id,
+                genesis_hash,
+                self.network.disable_discovery,
+                &self.bootnodes,
```

Comment:
As I wrote in another issue, we should prob start pulling these in the Config struct and keep the network_config function sig clean, only requiring a database?

Reply:
Similar to other issue, yeah moving the args would be cleaner.